src/util/mod.rs

```diff
@@ -241,6 +234,41 @@ pub fn unescape(s: &str) -> Vec<u8> {
     buf
 }
 
+#[cfg(test)]
+mod unescape_tests {
+    use super::*;
+
+    #[test]
+    fn test_no_escapes() {
+        assert_eq!(unescape(r"ab"), b"ab");
```

Comment:
Can them be merged together? We generally only keep one test function for the same function.

Reply:
Sure, I'll move it to the bottom tests module as well to keep it consistent.